wrangler/src/index.js

```diff
@@ -25,6 +26,7 @@ router
 .post('/tx-functions', txFunctionsUpload)
 .get('/tx-functions/:txFunctionHash', txFunctionsGet)
 .post('/tx-functions/:txFunctionHash', txFunctionsRun)
+.post('/tx-functions/:txFunctionHash/CB', txFunctionsRunCB)
```

Comment:
Is there a reason we are keeping the old stuff around on this branch?

We can recover from git if we want to revert. I think it will make this version quite odd to use with all the `sponsors` logic still existing alongside the new claimable balance method.

Reply:
Just backwards compatibility atm. Before merge we'd likely remove all the old txFunctionHash stuff, re-write the docs, etc. Just want to keep both routes open so people can safely deploy this change and test it "in prod" while we evaluate the change.